### PR TITLE
[PDP-1435] Change the Jfrog image repo for Infra charts

### PR DESCRIPTION
**Pull request by sasati-tibco**

### CODE REVIEW

**Review by pagarwal-tibco — CHANGES_REQUESTED** [review]
Please also make same changes for
artifact-manager
opentelemetry-collector
o11y-service
jaeger

These are also owned by our team and as part of the team is on leave from today. I request you to take care of them as well

File: charts/artifactmanager/templates/_helpers.tpl [hunk 1]
```diff
@@ -74,7 +74,7 @@ Integration storage folder pvc name
 
 {{- define "artifactmanager.sa" }}tp-dp-{{ .Values.global.cp.dataplaneId }}-sa{{ end -}}
 
-{{- define "artifactmanager.const.jfrogImageRepo" }}platform/integration{{end}}
+{{- define "artifactmanager.const.jfrogImageRepo" }}tibco-platform-local-docker/integration{{end}}
```
Comment (pagarwal-tibco) [inline charts/artifactmanager/templates/_helpers.tpl:77]: chart version update is missed

File: charts/o11y-service/templates/_helpers.tpl [hunk 2]
```diff
@@ -111,7 +111,7 @@ helm.sh/chart: {{ include "o11y-service.shared.labels.chartLabelValue" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion }}
 {{- end -}}
 
-{{- define "o11y-service.const.jfrogImageRepo" }}platform/infra{{end}}
+{{- define "o11y-service.const.jfrogImageRepo" }}tibco-platform-local-docker/infra{{end}}
```
Comment (pagarwal-tibco) [inline charts/o11y-service/templates/_helpers.tpl:114]: same

**Comment by sasati-tibco** [comment]
there will be one more PR for dp-core infra chart version change.

**Review by pagarwal-tibco — APPROVED** [review]

**Pull request merged**